pkg/manager/manager.go

```diff
@@ -38,6 +37,7 @@ import (
 	infrav1alpha4 "sigs.k8s.io/cluster-api-provider-vsphere/apis/v1alpha4"
 	infrav1 "sigs.k8s.io/cluster-api-provider-vsphere/apis/v1beta1"
 	vmwarev1 "sigs.k8s.io/cluster-api-provider-vsphere/apis/vmware/v1beta1"
+	topologyv1 "sigs.k8s.io/cluster-api-provider-vsphere/internal/apis/topology/v1alpha1"
```

Comment:
Looks like we have a few more imports of "github.com/vmware-tanzu/vm-operator/external/tanzu-topology/api/v1alpha1". Should we replace all of them?

Reply:
Make sense to me. Other imports are mostly in tests, I'll also replace them. This may be better for testing:-)